### PR TITLE
fix(ui-components): Update dialogs/callout border radius to follow latest VSCode

### PR DESCRIPTION
**Pull request by 815are**
Issue #1406 

Changes requested by @gpeuc
To be sync with VSCode updates/latest changes:
1. UIDialog, UICallout, UITooltip - border-radius 4px; https://github.com/SAP/open-ux-tools/pull/1407/commits/8e78d9ae3941ab8e83ca69b543df7aa20e6efb02
image

image

2. Dropdown menues and contextual menues - border-radius 2px; https://github.com/SAP/open-ux-tools/pull/1407/commits/5552aaf0a1ded6e27b5aad91b482a89584f9efc1
image

image

### CODE REVIEW

File: .changeset/sour-monkeys-joke.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/ui-components': patch
+---
+
+UIDialog, UICallout, UIContextualMenu, dropdown menues - apply border radius. 4px for UIDialog and UICallout. 2px for contextual and dropdown menues
```
Comment (devinea) [inline .changeset/sour-monkeys-joke.md:5]:
```suggestion
UIDialog, UICallout, UIContextualMenu, dropdown menus - apply border radius. 4px for UIDialog and UICallout. 2px for contextual and dropdown menus
```

**Review by devinea — COMMENTED** [review]
Code changes and tests look good. 
small typo in the changset.

File: .changeset/sour-monkeys-joke.md (hunk 1 above)
Comment (815are) [reply .changeset/sour-monkeys-joke.md]: thanks, applied - aa06765b4a6765bf993559bff8132818c64dc6a9

**Review by devinea — APPROVED** [review]
Thanks for the update

**Review by vadson71 — APPROVED** [review]
Changes look good to me. Didn't test them

**Pull request merged**